### PR TITLE
Add loki cost guide to handbook

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Add loki cost estimate guide to our public handbook to make it available to customers.

- Fix the aws-cost explorer images
- Move images to sub-folder

### CODE REVIEW

File: content/docs/observability/loki-cost-estimate.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+* The CPU & Memory consumption from all Loki's and Promtail's pods.
+* The object storage cost.
+* The data sent to the MC Loki by the promtail pods on WCs.
+
+The dashboard gives access to graphs related to those sources :
+
+![loki-cost-dashboard-screenshot](../images/loki-cost-dashboard.png)
+
+By basing yourself on the total storage space used by Loki to store the logs as well as the base cost of the object storage service from your cloud-provider, you can roughly estimate the cost of the object storage for your logging-infrastructure.
```
Comment (lyind) [inline content/docs/observability/loki-cost-estimate.md:29]:
```suggestion
Basing on the total storage space used by Loki to store the logs as well as the base cost of the object storage service from your cloud-provider, one can roughly estimate the cost of storage for the logging-infrastructure.
```

File: content/docs/observability/loki-cost-estimate.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+The same can be done for the data sent to Loki by the promtail pods. If you know the price for network transfer from your cloud-provider, you can have an estimation of the cost corresponding to this from the `Total of bytes transmitted over the network` Field of the dashboard.
```
Comment (lyind) [inline content/docs/observability/loki-cost-estimate.md:31]:
```suggestion
Similar applies to the data sent to Loki by the promtail pods. If one knows the price for network traffic from the cloud-provider, one can estimate the corresponding cost from the `Total of bytes transmitted over the network` field of the dashboard.
```

File: content/docs/observability/loki-cost-estimate.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Concerning the CPU and memory this gets a little more tricky but at least you can measure the increase in resources counsumption in your clusters.
```
Comment (lyind) [inline content/docs/observability/loki-cost-estimate.md:33]:
```suggestion
CPU and memory cost are a bit more tricky to quantify, but it's still possible to measure the increase in resource consumption in the clusters associated with Loki.
```

File: content/docs/observability/loki-cost-estimate.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+## Using Cost Explorer (AWS only)
+
+Alongside the dashboard mentioned above, if your logging-infrastructure is running on an AWS installation, you can open the AWS console and go to the Cost Explorer service. There, you can select the time period you wish to evaluate as well as the tag `giantswarm.io/installation` with the value corresponding to your installation's name :
```
Comment (lyind) [inline content/docs/observability/loki-cost-estimate.md:37]:
```suggestion
Alongside the dashboard mentioned above, if the logging infrastructure is running on an AWS installation, one can open the AWS console and navigate to the Cost Explorer service. There, select the time period to be evaluated as well as the tag `giantswarm.io/installation` with the value corresponding to the installation's name:
```

File: content/docs/observability/loki-cost-estimate.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+![cost-explorer-tag](../images/cost-explorer-tag.png)
+
+In addition to those parameters, you can also add more precision to the graph by grouping the result by services :
```
Comment (lyind) [inline content/docs/observability/loki-cost-estimate.md:41]:
```suggestion
In addition to those parameters, one can increase the graph's precision by grouping the result by services:
```

**Review by QuantumEnigmaa — APPROVED** [review]
LGTM

File: content/docs/observability/loki-cost-estimate.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+![cost-explorer-service](../images/cost-explorer-group-by.png)
+
+By selecting those parameters, you will get a similar looking graph :
```
Comment (lyind) [inline content/docs/observability/loki-cost-estimate.md:45]:
```suggestion
By selecting those parameters, one can generate a similar looking graph:
```

File: content/docs/observability/loki-cost-estimate.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+![cost-explorer](../images/aws-cost-explorer.png)
+
+You can then compare the general cost of your installation between a time period where the logging-infrastructure was not enabled and another one where it was.
```
Comment (lyind) [inline content/docs/observability/loki-cost-estimate.md:49]:
```suggestion
Using the graphs it's possible to compare the general cost of the installation between time periods with or without enabled logging infrastructure.
```

**Review by lyind — APPROVED** [review]
Minor style/spelling.

Feel free to ignore and only fix the few spelling/grammar mistakes contained in the suggestions.

**Comment by TheoBrigitte** [comment]
> Minor style/spelling.
> 
> Feel free to ignore and only fix the few spelling/grammar mistakes contained in the suggestions.
> 
> Other than that looks very good, nice graphics.

Thanks for the style/spelling.

**Pull request merged**